### ISSUE TITLE
Searching with --web results in a crash

### ISSUE THREAD

**Issue by paulofierro** [CocoaPods/CocoaPods#10866]
🌈
* [X ] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

### Command

```
/usr/local/bin/pod search Alamofire --web
```

### Report

* What did you do?

Attempted to do a search on cocoapods.org

* What did you expect to happen?

Get the search results.

* What happened instead?

An error occurred.

### Stack

```
   CocoaPods : 1.10.2
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin20]
    RubyGems : 3.0.3
        Host : macOS 11.5.2 (20G95)
       Xcode : 12.5 (12E262)
         Git : git version 2.30.1 (Apple Git-130)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : cocoapods - git - https://github.com/CocoaPods/Specs.git @ 76146f680503a1bf0ca02bf49d953556b3996ed9

               persona-id - git - git@github.com:persona-id/persona-podspecs.git @ e8a44aea87a9d9696fbbc9f6b0d006a24f019992

               trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.5.0
cocoapods-try         : 1.2.0
```

### Error

```
NoMethodError - undefined method `open!' for #<Pod::Command::Search:0x00007fb90b31b818>
Did you mean?  open
/Library/Ruby/Gems/2.6.0/gems/cocoapods-search-1.0.0/lib/cocoapods-search/command/search.rb:79:in `web_search'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-search-1.0.0/lib/cocoapods-search/command/search.rb:61:in `run'
/Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.2/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.2/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

**Comment by dnkoutso** [comment]
nice thanks for the report! transferring to cocoapods-search repo.

**Comment by dnkoutso** [comment]
Actually duplicate of https://github.com/CocoaPods/cocoapods-search/issues/24

**Comment by dnkoutso** [comment]
oh @paulofierro this is already fixed! https://github.com/CocoaPods/cocoapods-search/pull/26 just never released.

I will do a release of 1.0.1 with it.

**Comment by paulofierro** [comment]
@dnkoutso amazing, thank you 🎉

**Comment by dnkoutso** [comment]
let me fix its CI first with GH actions and clean it up and then will have 1.0.1 out.

**Comment by dnkoutso** [comment]
@paulofierro some good and bad news...I released 1.0.1 which does not crash anymore but realized the web url it creates it no longer works. I think its because our search.cocoapods.org repo uses algolia now...
